src/nt_users_tool/excel_processing.py

```diff
@@ -0,0 +1,82 @@
+from openpyxl import Workbook, load_workbook
+from datetime import date
+
+from constants import *
```

Comment:
Looks like you forgot this  `import *`  :)

Reply:
fixed in 04b3eac